### ISSUE TITLE
Label of images without any object

### ISSUE THREAD

**Issue by Adaverse** [AlexeyAB/darknet#4469]
Hey,
It would be really helpful if you could --help me with the label that is given to the images that have no object. I understand that we are supposed to keep it a blank txt file. I just want to know that labels is given to it while evaluating the loss

**Comment by AlexeyAB** [comment]
`delta_loss = 0 - output` if there is no object in this final activation https://github.com/AlexeyAB/darknet/blob/67e94dc1ded99b6d53637fa56e76e03731088e58/src/yolo_layer.c#L352

**Comment by 2akash111998** [comment]
Thank you! So only the objectness part of the loss function remains in the final loss. So is it safe to assume that leaving the txt file empty or giving it a coordinate of  '0.0 0.0 0.0 0.0' is the same thing.

**Comment by AlexeyAB** [comment]
> So is it safe to assume that leaving the txt file empty or giving it a coordinate of '0.0 0.0 0.0 0.0' is the same thing.

'0.0 0.0 0.0 0.0'  - isn't correct label, since there should be 5 values: `class_id x y w h`

----

So is it safe to assume that leaving the txt file empty or giving it a coordinate of '0 0.0 0.0 0.0 0.0' is the same thing.

Just because if turth_x==0 then this truth will be rejected.
